alignak/acknowledge.py

```diff
@@ -68,6 +69,7 @@ class Acknowledge(AlignakObject):  # pylint: disable=R0903
         'end_time': None,
         'author': None,
         'comment': None,
+        'comment_id': StringProp(default='')
```

Comment:
Why adding this parameter? And why making it a StringProp ... a string should be enough, no? Explain in the comments ...